app/src/main/java/com/appunite/loudius/network/datasource/AuthDataSource.kt

```diff
@@ -21,8 +21,6 @@ import com.appunite.loudius.network.model.AccessToken
 import com.appunite.loudius.network.model.AccessTokenResponse
 import com.appunite.loudius.network.services.AuthService
 import com.appunite.loudius.network.utils.WebException
-import javax.inject.Inject
-import javax.inject.Singleton
```

Comment:
Question: Is it really building in the correct way? We have annotations below and here are deleted imports ;)

Reply:
should be good now :)